Automattic/newspack-blocks
# Add .nvmrc; use LTS node

### All Submissions:






### Changes proposed in this Pull Request:



Adding [`.nvmrc`](https://github.com/nvm-sh/nvm#nvmrc) with `v10` node inside. 
Tried using `lts/*` (v12.13.1 at this point) but it throws a lot of `node-pre-gyp` errors. Staying at v10 to be safe here.

### How to test the changes in this Pull Request:

1. Checkout this branch
2. Run `nvm install` 
3. You should be now using the v10 node (`v10.17.0` at this point)
4. Nuke `node_modules` and run `npm install`
5. Run `npm start` to see if the build runs correctly
